kubevirt_scripts/Containerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.fedoraproject.org/fedora:39
+MAINTAINER Mario Vazquez <mavazque@redhat.com>
+
+RUN set -x && \
```

Comment:
you have 3 single RUN entries here, probably good to group them or even writing a script instead

Reply:
Grouped.